reaper/util.py

```diff
@@ -134,7 +134,7 @@ def create_archive(content, arcname, metadata=None, outdir=None):
         if metadata is not None:
             zf.comment = json.dumps(metadata, default=metadata_encoder)
         for fn, fp in files:
-            zf.write(fp, os.path.join(arcname, fn))
+            zf.write(fp, os.path.join(arcname, fn) if rootdir else fn)
```

Comment:
I found that the EEG viewer in the frontend _requires_ a zip where the .eeg and .vhdr are present at the root level (and that the zip filename ends with .eeg.zip). I assumed it was the decided format.